.github/ISSUE_TEMPLATE/work-item-issue.md

```diff
@@ -0,0 +1,81 @@
+# Work Item Issue Template
+
+## 1. **Title** *
+Provide a concise and descriptive title for the work item.
+
+Your input ...
+
+## 2. **Problem Statement** *
+Describe the problem, issue, or opportunity that this work item addresses.
+- **What is the problem?**: Explain the context and why it needs resolution.
+- **Impact**: Describe how the problem affects users, systems, or the project.
+
+Your input ...
+
+## 3. **Objective** *
+State the objective of this work item. What outcome is expected?
+- **Success Criteria**: Define how success will be measured (e.g., KPIs, metrics).
+
+Your input ...
+
+## 4. **Supporting Documentation** *
+Include links to supporting documents such as:
+- Data Examples: [Link to data or relevant files]
+- Related Use Cases or Discussion Documents: [Link to discussion]
+- PRs or Other References: [Link to relevant references]
+
+Your input ...
+
+## 5. **Proposed Solution / Approach**
+Outline any proposed solutions, approaches, or potential paths forward.
+- **Initial Ideas**: Describe potential solution paths, tools, or technologies.
+- **Considerations**: Include any constraints, dependencies, or risks.
+
+Your input ...
+
+## 6. **Epic or Theme Association**
```

Comment:
I don't think we should formally ask for this here.  There will often be issues created that do not yet have an Epic, and I don't necessarily want to think of Epics as a curated list from which the work item creator must choose.  Certainly, the maintainers should label with Theme / Epic as appropriate as part of downstream triage.

Reply:
I see this as a placeholder to add this information later. The field is optional, but we can enforce to don't add any details by indicating `to be completed by the Maintainers`

Reply:
@shawnalpay See later changes. I have added a comment indicating that the section will be completed by the Maintainers. What do you think?